### ISSUE TITLE
Add docs build to Travis-CI

### ISSUE THREAD

**Issue by stevepiercy** [Pylons/webtest#199]
It would be good to test that the docs build successfully on every commit. If you agree, then I can submit a PR with the changes to `tox.ini` and `.travis.yml`.

**Comment by gawel** [comment]
Of course. I don't know how to do that. I'll have a look at your changes

**Comment by stevepiercy** [comment]
I'm going to back out on this one. See the PR #198 for details and a history of my failed attempts.
